doc/reference/usb/index.rst

```diff
@@ -55,21 +55,32 @@ device controller driver and USB device class drivers or customer applications.
 It's a port of the LPCUSB device stack. It provides the following
 functionalities:
 
-   * Responds to standard device requests and returns standard descriptors,
-     essentially handling 'Chapter 9' processing, specifically the standard
-     device requests in table 9-3 from the universal serial bus specification
-     revision 2.0.
-   * Provides a programming interface to be used by USB device classes or
-     customer applications. The APIs are described in the usb_device.h file.
-   * Uses the APIs provided by the device controller drivers to interact with
-     the USB device controller.
+* Responds to standard device requests and returns standard descriptors,
+  essentially handling 'Chapter 9' processing, specifically the standard
+  device requests in table 9-3 from the universal serial bus specification
+  revision 2.0.
+* Provides a programming interface to be used by USB device classes or
+  customer applications. The APIs are described in the usb_device.h file.
+* Uses the APIs provided by the device controller drivers to interact with
+  the USB device controller.
 
 USB device class drivers
 ************************
 
-To initialize the device class driver instance the USB device class driver
-should call :c:func:`usb_set_config()` passing as parameter the instance's
-configuration structure.
+Zephyr USB Device Stack supports many standard classes, such as HID, MSC
+Ethernet over USB, DFU, Bluetooth.
+
+Implementing non standard USB class
+===================================
+
+Configuration of USB Device is done in the stack layer.
+
+Following structures and callbacks need to be defined:
+* Part of USB Descriptor table
```

Comment:
needs a blank line here before the first item in a list (otherwise the list is not recognized and just becomes part of the paragraph).